src/components/organisms/About/About.jsx

```diff
@@ -1,12 +1,25 @@
-import AboutImg from "../../molecules/AboutImg/AboutImg";
-import AboutText from "../../molecules/AboutText/AboutText"
 import "./About.scss";
 
 const About = (props) => {
-  return ( 
-    <div className="about"> 
-      <AboutImg image = {props.data.about.aboutImages}/>     
-      <AboutText aboutContent = {props.data.about}/>
+  return (
+    <div className="about">
+      <div className="about-image-container">
+        <img
+          className="about__image2"
+          src={props.data.about.aboutImages[0].path}
```

Comment:
Please send only necessary data to this component, do not send all data of the structure.